### ISSUE TITLE
[ERROR] Plugins quick-cmds can't download

### ISSUE THREAD

**Issue by FoyunW** [Eugeny/tabby#6725]
![image](https://user-images.githubusercontent.com/102196430/177243712-9e86e569-658e-483d-ae24-fe6fda54eddf.png)
When I click Get button always this error.

**Comment by rajubabannavar** [comment]
Yes I am also facing same issue with 1.0.181 in windows 10 installation

**Comment by Su11en** [comment]
Same problem with version 1.0.181 on Windows.

**Comment by huangsuper** [comment]
me too

**Comment by Yelijah** [comment]
Same problem with version 1.0.183 on macos

**Comment by finzzz** [comment]
I found a hotfix for this
```
podman run --rm -it -v $(pwd):/tmp docker.io/alpine:latest sh
apk add git npm
npm i tabby-quick-cmds
exit
```

This will generate folder called node_modules. Inside, there is a folder called `tabby-quick-cmds`
Just copy this and put that inside ~/.config/tabby/plugins/

**Comment by huangsuper** [comment]
> can u upload this mode?

**Comment by jy-tian** [comment]
> podman

Can you tell me how to do that
